ManagementPacks/Community.DataOnDemand.Unix/Scripts/ExtractUserFromProfile.ps1

```diff
@@ -0,0 +1,24 @@
+﻿# Extract username from a Unix/Linux run as profile prior to passing to the Microsoft.SystemCenter.WSManagement.Invoker
+Param(
+	$user
+)
+
+$scomAPI = New-Object -ComObject "MOM.ScriptAPI"
+$propertyBag = $scomAPI.CreatePropertyBag()
+
+if ([string]::IsNullOrEmpty($user))
+{
+	$propertyBag.AddValue("User", "")	
+}
+elseif ($user -match '<UserId>.+</UserId>')
+{
+	$modifiedUser = $user -replace '.*<UserId>([^<]+)</UserId>.*','$1'
```

Comment:
Instead of the <> characters?  Not to my knowledge since the tags of interested are managed by the SCOM GUI.